README.md

```diff
@@ -11,8 +11,8 @@ Create 3 promises:
 Add `success` and `error` handlers to each promise to show `<div data-qa="notification">` with `success` or `warning` class and a promise message.
 
 ## Instructions
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_promises_practice_DOM/)
+1. Replace `robsza1122` with your Github username in the link
+    - [DEMO LINK](https://robsza1122.github.io/js_promises_practice_DOM/)
```

Comment:
Add this link to the PR description
<img width="929" alt="image" src="https://github.com/mate-academy/js_promises_practice_DOM/assets/64750958/8f21c00a-9a82-484b-8d4d-ae86ca127d3e">